server/db/schema.ts

```diff
@@ -31,7 +31,17 @@ const ClassSchema = new Schema<ClassDocument>({
   classProfessors: { type: [String] }, // list of professors that have taught the course over past semesters
   classRating: { type: Number }, // the average class rating from reviews
   classWorkload: { type: Number }, // the average workload rating from reviews
-  classDifficulty: { type: Number } // the average difficulty rating from reviews
+  classDifficulty: { type: Number }, // the average difficulty rating from reviews
+  recommendations: [
+    {
+      _id: { type: String },
+      className: { type: String },
+      classSub: { type: String },
+      classNum: { type: Number },
+      tags: { type: [String] }, // array of tags, like ['overall rating', 'workload', 'difficulty']
```

Comment:
maybe this could be called recommendationTags just to make it more clear from the summaryTags?